index.md

```diff
@@ -0,0 +1,19 @@
+# Sumo Logic Helm Repository
+
+![Sumologic](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/images/sumo_logic_logo.png)
+
+## Add the Sumo Logic Helm repository
+
+```bash
+helm repo add sumologic https://sumologic.github.io/sumologic-kubernetes-collection
+```
+
+## Install Sumo Logic Helm Chart
+
+```bash
+helm upgrade --install my-release sumologic/sumologic --set sumologic.accessId=<SUMO_ACCESS_ID> --set sumologic.accessKey=<SUMO_ACCESS_KEY>  --set sumologic.clusterName="<MY_CLUSTER_NAME>"
+```
```

Comment:
```suggestion
```bash
helm upgrade --install my-release sumologic/sumologic \
    --set sumologic.accessId=<SUMO_ACCESS_ID> \
    --set sumologic.accessKey=<SUMO_ACCESS_KEY> \
    --set sumologic.clusterName="<MY_CLUSTER_NAME>"
```
```

Reply:
in addition add `"` to both `<SUMO_ACCESS_KEY>` and `<SUMO_ACCESS_ID>`

Reply:
Done:)